ui-tests/tests/widget.test.ts

```diff
@@ -79,6 +79,99 @@ test('test create new connection', async ({ page }) => {
 
 });
 
+
+const labels_in_memory = ['DuckDB', 'SQLite']
+
+
+test('test irrelevant fields in memory db', async ({ page }) => {
```

Comment:
make your test titles more descriptive, it took me some time to understand what we're testing here.

I believe you're checking that duckdb and sqlite do not show user, password, and host

a better title would be "test only relevant fields appear in embedded databases"

Reply:
[fixed](https://github.com/ploomber/jupysql-plugin/pull/73/commits/769403be5bc29b39d4c6d84c8a81607db59b7110)